src/datasources/alerts-api/tenderly-api.service.spec.ts

```diff
@@ -55,26 +51,35 @@ describe('TenderlyApi', () => {
         new TenderlyApi(
           fakeConfigurationService,
           mockNetworkService,
-          mockHttpErrorFactory,
+          httpErrorFactory,
         ),
     ).toThrow();
   });
 
   it('should add contracts', async () => {
+    const httpErrorFactoryFromSpy = jest.spyOn(httpErrorFactory, 'from');
```

Comment:
nitpick: I would say that we don't need to apply the `spyOn` for the scope of these tests.

We can work with the end result that is returned by the HttpErrorFactory.

Reply:
I've removed the spy and expectation in 8125503817eab4802917485d9c30ac52169a9040 as the response would otherwise throw, hence it's not needed.